crates/esthri/Cargo.toml

```diff
@@ -9,6 +9,7 @@ repository = "https://github.com/swift-nav/esthri/"
 homepage = "https://github.com/swift-nav/esthri/"
 readme = "../../README.md"
 keywords = ["aws", "s3"]
+rust-version = "1.56.1"
```

Comment:
we can also create a .clippy.toml file at the project root dir that specifies msrv = "1.56.1"

Reply:
Cool, feel free to submit a PR for this :-)